tools/testing/source_code_test.py

```diff
@@ -146,6 +146,7 @@ def test_no_experimental_api():
     # TODO: remove all elements of the list and remove the allowlist
     # This allowlist should not grow. Do not add elements to this list.
     allowlist = [
+        "tensorflow_addons/callbacks/average_model_checkpoint.py",
```

Comment:
I'm not familar with the policy here. @seanpmorgan Can you clarify if we can add items into this list.

Reply:
We shouldn't since we're still having trouble being compatible across TF releases. I'm not sure why this was added though? It doesn't look like any experimental APIs were added in this PR?

Reply:
Current api doesn't support loss scale optimizer. We could use stable api which is released in TF2.4 and is not compatible with tf2.3.

Reply:
IMO, loss scale optimizer is popular enough so we should support it. Actually, mixed precision is also a popular experimental feature and we use it in our tests ( bypassing this allowlist :-( ). Maybe hard restrictions on experimental APIs are not so comportable for us?

Reply:
@fsx950223 It's very sorry for the delay, but let's go through the whole process first before merging.